requirements.txt

```diff
@@ -5,6 +5,7 @@ click==6.7
 click-completion==0.3.1
 colorama==0.3.9
 cookiecutter==1.6.0
+docker>=2.0.0
```

Comment:
I'd accept it as extra, but not by default. This would have to be a separate discussion.

Also, there's two different dists: `docker` and `docker-py` implementing different API version. It may make sense to allow user to choose those.

P.S. Have you checked that they are well-compatible on the publish API level?